.github/workflows/checks.yml

```diff
@@ -28,5 +28,4 @@ jobs:
           node-version: 16.x
           cache: 'npm'
       - run: npm ci
-      - run: npx playwright install --with-deps
```

Comment:
if I understand correctly, it was needed only for running test:lib, so we can drop it